docs/tutorials/word-tutorial.md

```diff
@@ -0,0 +1,962 @@
+ms.date: 12/31/2018
+ms.topic: tutorial
+#Customer intent: As a developer, I want to build a Word add-in that can interact with content in a Word document.
+---
+
+# Tutorial: Create a Word task pane add-in
+
+In this tutorial, you'll create a Word task pane add-in that:
+
+> [!div class="checklist"]
+> * Inserts a range of text
+> * Formats text
+> * Replaces text and inserts text in various locations
+> * Inserts images, HTML, and tables
+> * Creates and updates content controls 
+
+## Prerequisites
+
+To use this tutorial, you need to have the following installed. 
+
+- Word 2016, version 1711 (Build 8730.1000 Click-to-Run) or later. You might need to be an Office Insider to get this version. For more information, see [Be an Office Insider](https://products.office.com/office-insider?tab=tab-1).
+
+- [Node](https://nodejs.org/en/) 
+
+- [Git Bash](https://git-scm.com/downloads) (or another Git client)
+
+## Create your add-in project
+
+Complete the following steps to create the Word add-in project that you'll use as the basis for this tutorial.
+
+1. Clone the GitHub repository [Word add-in tutorial](https://github.com/OfficeDev/Word-Add-in-Tutorial).
+
+2. Open a Git bash window, or Node.JS-enabled system prompt, and navigate to the **Start** folder of the project.
+
+3. Run the command `npm install` to install the tools and libraries listed in the package.json file. 
+
+4. Carry out the steps in [Adding Self-Signed Certificates as Trusted Root Certificate](https://github.com/OfficeDev/generator-office/blob/master/src/docs/ssl.md) to trust the certificate for your development computer's operating system.
+
+## Insert a range of text
+
+In this step of the tutorial, you'll programmatically test that your add-in supports the user's current version of Word, and then insert a paragraph into the document.
+
+### Code the add-in
+
+1. Open the project in your code editor.
+
+2. Open the file index.html.
+
+3. Replace the `TODO1` with the following markup:
+
+    ```html
+    <button class="ms-Button" id="insert-paragraph">Insert Paragraph</button>
+    ```
+
+4. Open the app.js file.
+
+5. Replace the `TODO1` with the following code. This code determines whether the user's version of Word supports a version of Word.js that includes all the APIs that are used in all the stages of this tutorial. In a production add-in, use the body of the conditional block to hide or disable the UI that would call unsupported APIs. This will enable the user to still use the parts of the add-in that are supported by their version of Word.
+
+    ```js
+    if (!Office.context.requirements.isSetSupported('WordApi', 1.3)) {
+        console.log('Sorry. The tutorial add-in uses Word.js APIs that are not available in your version of Office.');
+    }
+    ```
+
+6. Replace the `TODO2` with the following code:
```

Comment:
There's some inconsistencies in where we describe what the code blocks are doing. Sometimes it's in the numbered list, sometimes its in a note afterwards, and something there's nothing explicitly describing behavior.

Reply:
We'll plan to revisit this later when the content of this tutorial is re-evaluated / refactored.